packages/protvista-structure/README.md

```diff
@@ -1,13 +1,41 @@
 # `<protvista-structure>`
+
 In progress - A native web component wrapper around LiteMol
 
 ## Usage
+
 ```html
-      <protvista-structure id="1AAP"></protvista-structure>
+<!-- Basic usage -->
+<protvista-structure id="1AAP" />
```

Comment:
I don't think this would work on its own as it relies on the accession. Also `id` is not mentioned, I think that was an error in the doc.

Reply:
Actually I think it would be nice if we could bypass the `loadUniProtEntry()` if `accession` is not passed (which would also mean no table would be displayed).

Reply:
For the record: We won't be able to load the structure from PDBe call directly without knowing the protein accession we are interested in, because the PDBe call returns the structure details one level below the protein accession.